tasty-query/shared/src/main/scala/tastyquery/Types.scala

```diff
@@ -379,10 +389,14 @@ object Types {
 
     final def isRef(sym: Symbol)(using Context): Boolean =
       this match {
-        case tpe: NamedType   => tpe.symbol == sym
-        case tpe: AppliedType => tpe.underlying.isRef(sym)
-        case tpe: ExprType    => tpe.underlying.isRef(sym)
-        case _                => false // todo: add ProxyType (need to fill in implementations of underlying)
+        case tpe: NamedType    => tpe.symbol == sym
+        case tpe: AppliedType  => tpe.underlying.isRef(sym)
+        case tpe: ExprType     => tpe.underlying.isRef(sym)
+        case tpe: TermParamRef => tpe.underlying.isRef(sym)
+        case tpe: TypeParamRef =>
+          val bounds = tpe.bounds
+          bounds.low.isRef(sym) || bounds.high.isRef(sym)
```

Comment:
Shouldn't this be an `&&`? Or only test `bounds.high`?

Reply:
ive changed to only check `high`